### PR TITLE
User Guide: Update List Appointments

### LINKED ISSUE #17: User Guide: List Appointments
Update user guide with List Appointments commands

### PR DESCRIPTION
**Pull request by glatiuden**
- Updated the introduction of the UG
- Changed the Ui.png to the mockup Ui
- Modified the help window's URL to point to our user guide 
- Changed from `list` to `list_appointments`

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+Tutor Tracker is a **desktop app designed to help students search for tutors and manage tuition appointments**, optimised for use via a Command Line Interface (CLI) for a fast and streamlined experience.
```
Comment (kingsleykuan) [inline docs/UserGuide.md]: This update is no longer required as it was changed in another pull request, can revert to the current master revision.

**Review by kingsleykuan — CHANGES_REQUESTED** [review]
Looks good, however the UI image was already updated and is higher resolution than the one provided in this pull request. Either revert or keep the version from master during merging.

File: docs/UserGuide.md (hunk 1 above)
Comment (glatiuden) [reply docs/UserGuide.md]: Noted! Will omit during the merge.

**Review by kingsleykuan — DISMISSED** [review]

**Comment by glatiuden** [comment]
Resolved the merge conflicts and used the Ui image from the master branch instead.

**Review by jared98lyj — APPROVED** [review]

**Review by kingsleykuan — APPROVED** [review]

**Pull request merged**